src/pages/RFOX/components/Stake/StakeConfirm.tsx

```diff
@@ -170,10 +170,8 @@ export const StakeConfirm: React.FC<StakeConfirmProps & StakeRouteProps> = ({
   )
 
   const isApprovalTxPending = useMemo(
-    () =>
-      isApprovalMutationPending ||
-      (isApprovalMutationSuccess && approvalTx?.status !== TxStatus.Confirmed),
-    [approvalTx?.status, isApprovalMutationPending, isApprovalMutationSuccess],
+    () => isApprovalMutationPending || (isApprovalMutationSuccess && isApprovalRequired),
```

Comment:
Changes the heuristics to "we've broadcasted the Tx, but mutation is still pending"